src/utils/updateTokenPayloadToSingleToken.ts

```diff
@@ -3,24 +3,24 @@ import { SingleToken } from '@/types/tokens';
 
 export function updateTokenPayloadToSingleToken(
   payload: UpdateTokenPayload,
+  id?: string,
```

Comment:
Probably makes more sense to have 
```suggestion
  id: string = uuidv4(),
```
then we don't have to fix the old calls to this function

Reply:
if we should change the id as default parameter, then when we edit the token, tokenId will be changed. That is not what we expect.